extensions/multiple-references-extension/package.json

```diff
@@ -1,31 +1,50 @@
 {
-  "name": "field-editor-example",
+  "name": "multiple-references-extension",
+  "version": "1.0.0",
   "private": true,
+  "dependencies": {
+    "@contentful/app-sdk": "3.31.1",
+    "@contentful/field-editor-single-line": "^0.14.1",
+    "@contentful/forma-36-fcss": "^0.3.2",
+    "@contentful/forma-36-react-components": "^3.90.1",
+    "@contentful/forma-36-tokens": "^0.10.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
   "scripts": {
-    "start": "contentful-extension-scripts start",
-    "build": "contentful-extension-scripts build",
-    "deploy": "contentful-extension-scripts build && contentful extension update --force",
-    "configure": "contentful space use && contentful space environment use",
-    "login": "contentful login",
-    "logout": "contentful logout",
-    "help": "contentful-extension-scripts help"
+    "start": "cross-env SKIP_PREFLIGHT_CHECK=true BROWSER=none react-scripts start",
+    "build": "SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```

Comment:
```suggestion
    "build": "cross-env SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
```

Reply:
Good catch! I will add it to all extensions/apps in my next PR!